build.gradle

```diff
@@ -10,10 +10,11 @@ apply plugin: 'java-gradle-plugin'
 apply plugin: 'maven-publish'
 apply from: "$rootDir/gradle/functional-test.gradle"
 
+group = 'com.tomtom.gradle'
```

Comment:
Isn't the same duplicated in the `updateChangelog` section?

Reply:
It's true, but there's a catch. In the `updateChangelog` we have plugin ID, while this `group` specifies project's publication group. This `group` will be used for maven publication.
Let's re-use `group`  in other strings.